frontend/src/Editor/Inspector/ActionConfigurationPanels/SwitchPage.jsx

```diff
@@ -3,13 +3,15 @@ import Select from '@/_ui/Select';
 import defaultStyles from '@/_ui/Select/styles';
 import { CodeHinter } from '../../CodeBuilder/CodeHinter';
 import { useTranslation } from 'react-i18next';
+import { useCurrentState } from '@/_stores/currentStateStore';
 
 export function SwitchPage({ getPages, event, handlerChanged, eventIndex, darkMode }) {
   const queryParamChangeHandler = (index, key, value) => {
     event.queryParams[index][key] = value;
     handlerChanged(eventIndex, 'queryParams', event.queryParams);
   };
   const { t } = useTranslation();
+  const currentState = useCurrentState();
```

Comment:
Seems like currentstate is not used in this file

Reply:
Yes, removed it.